src/components/PageHero.tsx

```diff
@@ -0,0 +1,108 @@
+import styled from '@emotion/styled';
+
+import { FC } from 'react';
+import { Heading } from '~/components';
+import {
+  atMinLg,
+  atMinDesktop,
+  atMinLargeDesktop,
+  bp,
+  BreakpointName,
+} from '~/theme/breakpoints';
+import { cssClamp } from '~/theme/util';
+
+type HeroImageSource = {
+  largeDesktop: string;
+  tablet: string;
+  xl: string;
+  xs: string;
+};
+
+type HeroImageProps = {
+  alt: string;
+  imgSource: HeroImageSource;
+};
+
+type PageHeroProps = HeroImageProps & {
+  text: string;
+};
+
+const calculatedImageHeight = cssClamp(
+  [17.5, 'smMobile'],
+  [20.188, 'mobile'],
+  [35, 'tablet'],
+);
+
+const Image = styled.picture`
+  margin-left: calc(50% - 50vw);
+
+  ${atMinLargeDesktop} {
+    margin-left: ${cssClamp([-20, 'largeDesktop'], [-20, 'xlDesktop'])};
+  }
+
+  img {
+    filter: brightness(0.85);
+    height: ${calculatedImageHeight};
+    position: absolute;
+  }
+`;
+
+const HeaderText = styled(Heading)`
+  margin-top: ${cssClamp([15, 'xs'], [20, 'tablet'])};
```

Comment:
🔧 🎨 At 375 the text appears a bit lower than it does in the comps. because the clamp calc starting at xs doesn't quite hit right.

I think that this looks a bit closer
```suggestion
  margin-top: ${cssClamp([12.5, 'smMobile'], [15, 'mobile'], [20, 'tablet'])};
```

Reply:
Whoops, thanks Tony!